packages/aws-s3/aws-s3.1.1.1/opam

```diff
@@ -0,0 +1,22 @@
+opam-version: "1.2"
+maintainer: "Anders Fugmann <anders@fugmann.net>"
+authors: "Anders Fugmann"
+license: "BSD-3"
+homepage: "https://github.com/andersfugmann/aws-s3"
+dev-repo: "git+https://github.com/andersfugmann/aws-s3"
+bug-reports: "https://github.com/andersfugmann/aws-s3/issues"
+build: ["jbuilder" "build" "-p" name "-j" jobs]
+depends: [
+  "jbuilder" { build }
+  "core" { >= "0.9" }
```

Comment:
this bound should be `>="v0.9"` (note the "v" character)

Reply:
Nice spotted. I have added the missing 'v'.